x-pack/legacy/plugins/maps/public/components/layer_toc_actions.js

```diff
@@ -51,7 +52,11 @@ export class LayerTocActions extends Component {
   };
 
   _renderPopoverToggleButton() {
-    const { icon, tooltipContent } = this.props.layer.getIconAndTooltipContent(this.props.zoom);
+    const { icon, tooltipContent, areResultsTrimmed } = this.props.layer.getIconAndTooltipContent(this.props.zoom);
+    const infoButton = areResultsTrimmed ? (<EuiIconTip
```

Comment:
This should just be EuiIcon because you're not using it as a tooltip itself. Also, just make it subdued so you're not making the whole label transparent.
```suggestion
    const infoButton = areResultsTrimmed ? (<EuiIcon color="subdued"
```